js/combobox.js

```diff
@@ -13,10 +13,7 @@ $(function() {
          this.input.keydown(function(event) {
            var enterkey = 13;
            if(event.keyCode === enterkey) {
-             var $firstMenuItem = $($('li')[0]);
-             if ($firstMenuItem.is(':visible')) {
-               $firstMenuItem.click();
-             }
+             $(".ui-autocomplete").find("li:first:visible").click();
```

Comment:
Since `:first` and `:filter` are not part of CSS, I would instead write this as:

```javascript
$(".ui-autocomplete").find("li").filter(":first:visible").click();
```

in order to clarify/separate the CSS locators from the jQuery filters. But that's my personal preference, please do what you think is best.